fendermint/eth/api/Cargo.toml

```diff
@@ -24,6 +24,7 @@ tracing = { workspace = true }
 tendermint = { workspace = true }
 tendermint-rpc = { workspace = true }
 tokio = { workspace = true }
+reqwest = { workspace = true }
```

Comment:
Can this just be among `[dev-dependencies]`?

Reply:
Agree. Fortunately we don't even need it per your comment above.